pythonforandroid/recipes/python3/patches/remove-fix-cortex-a8.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/configure b/configure
```

Comment:
It would be great that you put a comment in the top of the patch, explaining why we need these patch, because that will make easier to maintain it in a future for all of us :wink:

**Also see:** https://stackoverflow.com/questions/18979120/is-it-possible-to-add-a-comment-to-a-diff-file-unified